cmd/runmqserver/main.go

```diff
@@ -109,12 +109,22 @@ func doMain() error {
 		logTermination(err)
 		return err
 	}
+
 	err = createDirStructure()
 	if err != nil {
 		logTermination(err)
 		return err
 	}
 
+	// handle /var/mqm/ permissions in upgrade to v4.0.0
```

Comment:
v4.0.0 is the version of the Helm chart, which may be confusing here

Reply:
Updated to `upgrade to ubi`